nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
Just tried. `self._shape` is a tuple, so that doesn't work. (Failed to try the right tests, locally. Want `test_proxy_api`, if you're playing along at home.)